README.md

```diff
@@ -37,11 +37,13 @@ map each to the appropriate type, as shown below.
 // db-refs.ts
 import {
   collection,
-  type CollectionReference,
+  FirebaseFirestoreTypes,
 } from "@react-native-firebase/firestore";
 import { db } from "./firestore";
 import { User, WishlistItem, Book } from "./types";
 
+type CollectionReference<T extends FirebaseFirestoreTypes.DocumentData> = FirebaseFirestoreTypes.CollectionReference<T>;
```

Comment:
We are now re-exporting the firebase types from the library, so please replace this with 

```
// Firestore types are available here in a more convenient format.
import type { CollectionReference } from "@typed-firestore/react-native";
```